### PR TITLE
organized resource folder, created and updated player class

### PR DESCRIPTION
**Pull request by hundooo**

### CODE REVIEW

File: src/objects/Player.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package objects;
+
+import java.util.Arrays;
+
+public class Player {
+	private Card[] hand;
```
Comment (itslinotlie) [inline src/objects/Player.java]: You are missing a boolean array to check if the cards are completed or not

File: src/objects/Player.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	private int playerID;
```
Comment (itslinotlie) [inline src/objects/Player.java]: Since we have a player array (in the GUI), having a playerID field seems redundant

**Review by itslinotlie — CHANGES_REQUESTED** [review]
We just need to make sure we init the Deck instance somewhere in the code before using it

**Comment by itslinotlie** [comment]
And reminder for next time, if you plan on implementing unique features, it might be a good idea to work in branches so that different features don't end up on the same PR

File: src/objects/Player.java (hunk 2 above)
Comment (hundooo) [reply src/objects/Player.java]: done.

File: src/objects/Player.java (hunk 1 above)
Comment (hundooo) [reply src/objects/Player.java]: done.

**Review by SheepStar02 — APPROVED** [review]

**Pull request merged**